### PR TITLE
set-output is deprecrated and will soon been removed

### PR DESCRIPTION
**Pull request by jmazzitelli**
fixes: https://github.com/kiali/openshift-servicemesh-plugin/issues/193

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -20,13 +20,13 @@ jobs:
     name: Initialize
     runs-on: ubuntu-20.04
     outputs:
-      target_branch: ${{ steps.target_branch.outputs.target_branch }}
-      build-branch: ${{ steps.build_branch.outputs.build_branch || steps.branch-pr.outputs.branch-pr }}
+      target_branch: ${{ env.target_branch }}
+      build_branch: ${{ env.build_branch || env.branch-pr }}
```
Comment (jmazzitelli) [inline .github/workflows/ci.yaml:24]: I don't see `branch-pr` used anywhere - I think this is old, obsolete code. I think `|| env.branch-pr` should be removed.

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -165,44 +165,44 @@ jobs:
...
 
-        echo "Plugin Quay tag": ${{ steps.quay_tag.outputs.plugin_quay_tag }}
+        echo "Plugin Quay tag: ${{ env.plugin_quay_tag }}"
 
-        echo "Operator Quay tag": ${{ steps.quay_tag.outputs.operator_quay_tag }}
+        echo "Operator Quay tag: ${{ env.operator_quay_tag }}"
 
   release:
     name: Release
-    if: ${{ needs.initialize.outputs.release_type != 'skip' && ((github.event_name == 'schedule' && github.repository == 'kiali/openshift-servicemesh-plugin') || github.event_name != 'schedule') }}
```
Comment (jmazzitelli) [inline .github/workflows/release.yaml:196]: I'm not sure if `needs.initialize.output` can be replaced with env. @leandroberetta ?

**Review by jmazzitelli — COMMENTED** [review]
@leandroberetta I gave it a shot. Does this look OK? I didn't run any of it yet.

File: .github/workflows/release.yaml (hunk 2 above)
Comment (jmazzitelli) [reply .github/workflows/release.yaml:196]: nm.. I need to keep this the way it was

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (jmazzitelli) [reply .github/workflows/ci.yaml:24]: I'm removing branch-pr - this isn't set anywhere. I think this is an old remnant that is no longer used/set.

**Comment by jmazzitelli** [comment]
CI workflow works... I see it pass on my fork repo.
Testing nightly and release now... if you see new images on my quay repo, you'll know those tests pass too:
* https://quay.io/repository/jmazzitelli/ossmconsole?tab=tags
* https://quay.io/repository/jmazzitelli/ossmconsole-operator?tab=tags
See my fork's actions: https://github.com/jmazzitelli/openshift-servicemesh-plugin/actions

**Comment by jmazzitelli** [comment]
Ready for review and approval - all my tests show that this works.

**Review by leandroberetta — APPROVED** [review]
Looks good.

**Pull request merged**